src/main/java/io/github/railroad/debugger/syntax/SyntaxHandler.java

```diff
@@ -31,6 +30,7 @@ public static void main(String[] args) {
 	private ExecutorService executor;
 
 	@SuppressWarnings("unused")
+	// @SuppressWarnings Should be removed, usually if you have to use SuppressWarnings, you have something to fix :)
```

Comment:
I feel like this is obvious? If it's unused then we know we have something to fix. We are just removing a stupid warning from our project.

Reply:
I'm not trying to say it's bad or anything, just wanted to point it out 😛